docs/overview.rst

```diff
@@ -44,5 +50,5 @@ and perhaps a high-level I/O library, like HDF5, ADIOS, or PnetCDF.
 A UnifyFS server process runs on each compute node in
 the job allocation. The UnifyFS server handles the I/O
 requests from the UnifyFS library.
-The UnifyFS server uses ECP `Mochi <https://mochi.readthedocs.io/en/latest>`_
+The UnifyFS server uses ECP `Mochi <https://mochi.readthedocs.io/en/latest>`
```

Comment:
Same here. I think the underscore is needed here for external hyperlinks to work.